src/security-server/system-test/src/intTest/resources/compose.systemtest.yaml

```diff
@@ -49,7 +51,7 @@ services:
       - ../../../src/security-server/system-test/src/intTest/resources/container-files/etc/xroad/backup-keys:/etc/xroad/backup-keys
 
   db-opmonitor:
-    image: postgres:17
+    image: postgres:16
```

Comment:
any particular reason to downgrade?

Reply:
To match with pg_dump version used in backup-manager application. ubuntu 24 has postgresql v16.